### PR TITLE
Fixed up lines (loosely) related to Kvailas Forest

### PR DESCRIPTION
**Pull request by Ihysoal**
Corrected a severely mistranslated line
Adjusted several dialogue lines based around the Kvailas Forest quest chain.
Unified monster race identifier to Merog
Edited dialogue using Merog in its lines.
Changed all entries referring to "Fallen Worry Forest" to "the Fallen Worry Forest"
Edited dialogue using Fallen Worry Forest in its lines.
Edited lines containing Bramble to indicate he is a single entity.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -2552,7 +2552,7 @@ ETC_20150317_002551	Naktis' subordinates are being too disrupting!
 ETC_20150317_002552	Using a key of Mayvern..
 ETC_20150317_002553	Experimental Mesyum
 ETC_20150317_002554	Acidic solution Maki
-ETC_20150317_002555	While using to change the horsepower
+ETC_20150317_002555	Using the Refined Spell Crystal
```
Comment (Sunaries) [inline ETC.tsv]: Magic Crystal.

File: ETC.tsv [hunk 2]
```diff
@@ -7889,7 +7889,7 @@ ETC_20150317_007889	Capture the suspicious movement (4)
 ETC_20150317_007890	Capture the suspicious movement (5)
 ETC_20150317_007891	Surprise Attack of Maegro
 ETC_20150317_007892	Load of Bramble, Gaigalas
-ETC_20150317_007893	Create a enhanced thorn pollen stimulant to mix with a Maegro's Mspittle and a Pollen stimulant
+ETC_20150317_007893	Create an enhanced thorn pollen stimulant by mixing it with a Merog's saliva and a pollen stimulant
```
Comment (Sunaries) [inline ETC.tsv]: a Merog's saliva -> Merog's saliva

File: ITEM.tsv [hunk 3]
```diff
@@ -3353,12 +3353,12 @@ ITEM_20150317_003352	Revelator's Proof
...
 ITEM_20150317_003354	Spatial Crest 
 ITEM_20150317_003355	A special key for finding revelations and hidden places. 
-ITEM_20150317_003356	Merog Spittle 
+ITEM_20150317_003356	Merog Saliva 
 ITEM_20150317_003357	It creates an awakening effect if it is mixed with Thorn Flower Powder Stimulant. 
 ITEM_20150317_003358	Enhanced Thorn Flower Stimulant 
 ITEM_20150317_003359	It has a better effect than the Thorn Flower Powder  
 ITEM_20150317_003360	Thorn Flower Fluid 
-ITEM_20150317_003361	It has an awakening effect if mixed with a Merog's spit. 
+ITEM_20150317_003361	It has an awakening effect if mixed with a Merog's saliva.
```
Comment (Sunaries) [inline ITEM.tsv]: a Merog's saliva -> Merog's saliva.

File: ITEM.tsv [hunk 4]
```diff
@@ -3373,7 +3373,7 @@ ITEM_20150317_003372	A pot filled with the magical energy that flows through the
 ITEM_20150317_003373	Black Maize Poison  
 ITEM_20150317_003374	Ingredient necessary to redecorate the Obelisk. It has a high level of acidity. 
 ITEM_20150317_003375	Paralysis Powder 
-ITEM_20150317_003376	Paralysis Powder that can stun the Black Maize in Rhombuspaving Dale. 
+ITEM_20150317_003376	Paralysis Powder that can stun the Black Maize in Rhombuspaving Dale. I
```
Comment (Sunaries) [inline ITEM.tsv]: I? (end of the sentence)

File: QUEST_JOBSTEP.tsv [hunk 5]
```diff
@@ -103,9 +103,9 @@ QUEST_JOBSTEP_20150317_000102	I'm surprised you found it so fast! {nl}And it's i
 QUEST_JOBSTEP_20150317_000103	I think you might be able to buy the next book from the Hunter Master..{nl}I'm broke after fixing the wall with my money. Haha..
 QUEST_JOBSTEP_20150317_000104	You mean the second book of Lydia Schaffen? {nl}I'll hand it over for 5000 silver. 
 QUEST_JOBSTEP_20150317_000105	I think around 5000 silver will do. {nl}I heard that's the book's value. 
-QUEST_JOBSTEP_20150317_000106	Ah, you came at the right time.{nl}If you could take the leathers of Meroggs to Quarrel Shooter Master, he will make you a copy of the next volume.
-QUEST_JOBSTEP_20150317_000107	Did the leathers of Merogg come from Archer Master?
-QUEST_JOBSTEP_20150317_000108	You will see Meroggs easily at Fallen Worry Forest.{nl}Take those to Quarrel Shooter Master in Klaipeda.
+QUEST_JOBSTEP_20150317_000106	Ah, you came at the right time.{nl}If you could take this Merog leather to the Quarrel Shooter Master, he will make you a copy of the next volume.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv:106]: You capitalized Leather in an earlier line.

File: QUEST_JOBSTEP.tsv [hunk 6]
```diff
@@ -103,9 +103,9 @@ QUEST_JOBSTEP_20150317_000102	I'm surprised you found it so fast! {nl}And it's i
...
+QUEST_JOBSTEP_20150317_000107	Did this Merog leather come from the Archer Master?
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv:107]: You capitalized Leather in an earlier line.

File: QUEST_JOBSTEP.tsv [hunk 7]
```diff
@@ -117,9 +117,9 @@ QUEST_JOBSTEP_20150317_000116	Oh, that's pretty good. {nl}It will he helpful for
 QUEST_JOBSTEP_20150317_000117	The problem is the Hunter Master. I really don't want to see her face. {nl}So you go and get the next book from her. 
 QUEST_JOBSTEP_20150317_000118	You mean the book I have? {nl}Well, okay. I'll hand it over for 5000 silver. 
 QUEST_JOBSTEP_20150317_000119	I think around 5000 silver will do. {nl}Can you spare that much? 
-QUEST_JOBSTEP_20150317_000120	You came at the right time.{nl}Quarrel Shooter Master told you that if you could bring the leathers of Meroggs, he will make you a copy of the next volume.
-QUEST_JOBSTEP_20150317_000121	Didn't Ranger Master tell you anything?{nl}I asked him for Meroggs' leathers.
-QUEST_JOBSTEP_20150317_000122	Meroggs are at Fallen Worry Forest and Quarrel Shooter Master is at Klaipeda. {nl}Someone may go there and get the book faster than you so please hurry.
+QUEST_JOBSTEP_20150317_000120	You came at the right time.{nl}Quarrel Shooter Master told you that if you could bring the leathers of Merogs, he will make you a copy of the next volume.
+QUEST_JOBSTEP_20150317_000121	Didn't the Ranger Master tell you anything?{nl}I asked him for Merog leather.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv:121]: You capitalized Leather in an earlier line.

File: QUEST_JOBSTEP.tsv [hunk 8]
```diff
@@ -1275,11 +1275,11 @@ QUEST_JOBSTEP_20150317_001274	Give the book to Hunter Master
 QUEST_JOBSTEP_20150317_001275	Foudn the book Hunter Master is looking for. Give it to Hunter Master.
 QUEST_JOBSTEP_20150317_001276	Killed Vubbe Fighter and acquired %s
 QUEST_JOBSTEP_20150317_001277	This isn't the only book the Hunter Master is looking for. Talk to him again.
-QUEST_JOBSTEP_20150317_001278	Collect the leathers of Merogg
-QUEST_JOBSTEP_20150317_001279	Hunter Master told you that you may be able to trade with Quarrel Shooter Master with the leathers of Merogg you have for the book. Obtain the leathers of Meroggs from Fallen Worry Forest.
-QUEST_JOBSTEP_20150317_001280	Hand over the leathers of Merogg to Quarrel Shooter Master
-QUEST_JOBSTEP_20150317_001281	You've obtained enough Leathers of Meroggs that are needed by Quarrel Shooter Master. Hand over the leathers of Meroggs to Quarrel Shooter Master at Klaipeda.
-QUEST_JOBSTEP_20150317_001282	Obtain %s by defeating Meroggs
+QUEST_JOBSTEP_20150317_001278	Collect the leathers of Merog
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: of _or_ from Merog?

File: QUEST_JOBSTEP.tsv [hunk 9]
```diff
@@ -1275,11 +1275,11 @@ QUEST_JOBSTEP_20150317_001274	Give the book to Hunter Master
...
+QUEST_JOBSTEP_20150317_001279	Hunter Master told you that you may be able to trade with Quarrel Shooter Master with the leathers of Merog you have for the book. Obtain the leather of Merogs from the Fallen Worry forest.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: of _or_ from Merog?

File: QUEST_JOBSTEP.tsv [hunk 10]
```diff
@@ -1275,11 +1275,11 @@ QUEST_JOBSTEP_20150317_001274	Give the book to Hunter Master
...
+QUEST_JOBSTEP_20150317_001280	Hand over the leathers of Merog to Quarrel Shooter Master
+QUEST_JOBSTEP_20150317_001281	You've obtained enough Leathers of Merogs that are needed by Quarrel Shooter Master. Hand over the leathers of Merogs to Quarrel Shooter Master at Klaipeda.
+QUEST_JOBSTEP_20150317_001282	Obtain %s by defeating Merogs
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv:1282]: Merogs -> Merog 

I assume it says: "Obtain Merog Leather" by defeating ...

File: QUEST_JOBSTEP.tsv [hunk 11]
```diff
@@ -1402,12 +1402,12 @@ QUEST_JOBSTEP_20150317_001401	Defeat the body of evil spirit
...
 QUEST_JOBSTEP_20150317_001404	Collect the materials for Wizard Master
-QUEST_JOBSTEP_20150317_001405	Collect the Skins of Grolls in Spine Heart Dale, Feelers of Chafperors in Fallen Worry Forest, and the nucleuses of Yukopuses from Siaulai Mine Village.
+QUEST_JOBSTEP_20150317_001405	Collect the Skins of Grolls in Spine Heart Dale, Feelers of Chafperors in the Fallen Worry forest, and the nucleuses of Yukopuses from Siaulai Mine Village.
 QUEST_JOBSTEP_20150317_001406	Give the materials to Wizard Master
 QUEST_JOBSTEP_20150317_001407	Collected the materials for Wizard Master. Give it to him.
 QUEST_JOBSTEP_20150317_001408	Collect %s from Spine Heart Dale
-QUEST_JOBSTEP_20150317_001409	Collect %s from Fallen Worry Forest
-QUEST_JOBSTEP_20150317_001410	Collect %s from Mine Village
+QUEST_JOBSTEP_20150317_001409	Collect %s from the Fallen Worry forest
+QUEST_JOBSTEP_20150317_001410	Collect %s from Siaulai Mine Village
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Remove Siaulai and change it to Miners' Village. It is true that in another sentence it says Siaulai Miners' Village. Inconsistency on IMC's part?

File: ITEM.tsv (hunk 4 above)
Comment (Ihysoal) [inline ITEM.tsv]: That's fishy, how did that get in?

File: QUEST_JOBSTEP.tsv (hunk 11 above)
Comment (Ihysoal) [inline QUEST_JOBSTEP.tsv]: Hmm, but we could just keep it like this? It's inconsistent either way. Oh well, I'll change it back.

**Comment by Ihysoal** [comment]
fixes*
God damn it's too late here.

Although Sunaries/Spadow might not entirely agree with some of the changes, it's unavoidable until guidelines are established.

File: QUEST_LV_0100.tsv [hunk 12]
```diff
@@ -2047,18 +2047,18 @@ QUEST_LV_0100_20150317_002046	Have you seen my ring?{nl}If you see that ring, pl
...
+QUEST_LV_0100_20150317_002053	I don't know if it's due to the thorny bushes or the vicious atmosphere, but {nl}this forest makes people depressed.
 QUEST_LV_0100_20150317_002054	I believe in myself that one day this forest will be okay again.
 QUEST_LV_0100_20150317_002055	Are you okay?{nl}I almost lost my conciousness due to that voice.{nl}Who is he anyways..
 QUEST_LV_0100_20150317_002056	
-QUEST_LV_0100_20150317_002057	The atmosphere is becoming stronger and the priests are getting tired. It's bad.
-QUEST_LV_0100_20150317_002058	Thanks.{nl}I wonder what would have happened without you.
-QUEST_LV_0100_20150317_002059	As Merogg sorcerers are undertaking their rituals, I will give it a try.{nl}Can you take care of those sorcerers?
+QUEST_LV_0100_20150317_002057	This is bad, the atmosphere is becoming stronger and the priests are getting tired.
+QUEST_LV_0100_20150317_002058	Thanks.{nl}I'm not sure what would've happened without you.
+QUEST_LV_0100_20150317_002059	Merog shamans are undertaking a ritual.{nl}Can you take care of them?
```
Comment (vtange) [inline QUEST_LV_0100.tsv:2059]: pretty sure my PR contains this line, merge conflict possible

File: QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -2070,23 +2070,23 @@ QUEST_LV_0100_20150317_002069
...
 QUEST_LV_0100_20150317_002085	I can rely on you.{nl}Please protect the altar until the fountain gets purified.
-QUEST_LV_0100_20150317_002086	Put Merlog's saliva in this potion and shake it. {nl} It will help you withstand the muddy aura of Bramble.
-QUEST_LV_0100_20150317_002087	Bramble is recovering by embedding roots around the Thorn Forest . {nl} Cutting those roots will be very painful.
-QUEST_LV_0100_20150317_002088	You can't let your guards down even if the Brambles weakened. {nl} It is the leader of the demons no matter how injured it is.
-QUEST_LV_0100_20150317_002089	Now we'll make a full-scale attack on Bramble. {nl} I'll go in the Giliaii courtyard and watch on the Brambles first so follow me.
+QUEST_LV_0100_20150317_002086	Put Merog saliva in this potion and shake it.{nl}It will help you withstand the muddy aura of Bramble.
+QUEST_LV_0100_20150317_002087	Bramble is recovering by embedding roots around the Thorn Forest.{nl}Cutting those roots will hurt him.
+QUEST_LV_0100_20150317_002088	You can't let your guards down even if Bramble is weakened.{nl}They are one the leader of the demons after all.
```
Comment (vtange) [inline QUEST_LV_0100.tsv:2088]: this and 2087 also will conflict with mine

File: QUEST_LV_0100.tsv [hunk 14]
```diff
@@ -6851,7 +6851,7 @@ QUEST_LV_0100_20150428_006850	The assistant commander Volda asked you to get rid
 QUEST_LV_0100_20150428_006851	Find the person who knows about the epitaph of Gailla Flurry and listen to his request.
 QUEST_LV_0100_20150428_006852	You found the luggage of the villager. Return the luggage to the villager and ask him about the contents in the epitaph.
 QUEST_LV_0100_20150428_006853	Defeat Blue Woodspirits that appeared near Obelisk
-QUEST_LV_0100_20150428_006854	Siaulai Goddess lost the revelation from Brambles at the thorny forest of Kbailas forest. Help the priests at Gate Route and Spine Heart Dale and look for the revelation at Kbailas forest.
+QUEST_LV_0100_20150428_006854	Siaulai Goddess lost the revelation from Bramble at the thorny forest of Kbailas forest. {nl}Help the priests at Gate Route and Spine Heart Dale and look for the revelation at Kbailas forest.
```
Comment (vtange) [inline QUEST_LV_0100.tsv:6854]: Another possible conflict line

**Comment by vtange** [comment]
Can this PR hold? I have one that rolls through the same general area. I like how you standardized the Merog name, but there are a lot of NPC stuff I did real time that I'd like not to lose

**Comment by Ihysoal** [comment]
Hmpf. Just resolve the merge conflicts and let vtange override me.

**Comment by vtange** [comment]
I just went through my own QUEST_lv100 to add the Merog fix and other lines you made. You can remove that file from your PR now to avoid conflicts

**Pull request closed without merging**